docs/workflows/create_sync_publish.rst

```diff
@@ -112,6 +112,34 @@ Assuming your token is ``YourRepositoryToken``, create the remote with the ``sle
     policy='on_demand' \
     sles_auth_token='YourRepositoryToken'
 
+.. _create-uln-remote:
+
+Creating an ULN remote
+**********************
+
+ULN stands for "Unbreakable Linux Network" and refers to the way Oracle in particular, `delivers Oracle Linux repositories <https://linux.oracle.com/>`_ to their enterprise customers.
```

Comment:
I am unsure if embedding external resources in the text like this is desired. Please advise.
An alternative might be to collect external resources in a bullet list at the end of the section, or else omit them entirely.

Reply:
In my opinion it's ok, as is.